docker-compose.yml

```diff
@@ -8,6 +8,7 @@ services:
       - "4983:4983"
     volumes:
       - ./apps/web:/app/apps/web
+      - ./packages/database:/app/packages/database
```

Comment:
This do the hot reload magic.

Reply:
I'm not so happy with this. I'm trying to have everything dockerized but if this packages lived in another repo `pnpm link` might not work. I have to test it.

Reply:
I think it's wrong to think of database as another npm package, this is really just shared code between api and web, it will never live in another repo and i'd just bundle it with the production builds of web and api.